### PR TITLE
add antlr visitor for MySQL TCLStatement.

### PR DESCRIPTION
**Pull request by dongzl**
For #3914 .

Changes proposed in this pull request:
- add antlr visitor for MySQL TCLStatement.

### CODE REVIEW

File: shardingsphere-sql-parser/shardingsphere-sql-parser-mysql/src/main/java/org/apache/shardingsphere/sql/parser/MySQLVisitor.java [hunk 1]
```diff
@@ -311,6 +327,31 @@ public ASTNode visitSetAutoCommit(final SetAutoCommitContext ctx) {
         return result;
     }
     
+    @Override
+    public ASTNode visitAutoCommitValue(final AutoCommitValueContext ctx) {
```
Comment (tristaZero) [inline shardingsphere-sql-parser/shardingsphere-sql-parser-mysql/src/main/java/org/apache/shardingsphere/sql/parser/MySQLVisitor.java]: Hi `AutoCommitSegment` is necessary for `SetAutoCommitStatement`

Comment (dongzl) [reply shardingsphere-sql-parser/shardingsphere-sql-parser-mysql/src/main/java/org/apache/shardingsphere/sql/parser/MySQLVisitor.java]: done.

**Comment by dongzl** [comment]
close this pr directly, submit a new pr on master branch.

**Comment by dongzl** [comment]
#4106 .

**Pull request closed without merging**